pkg/vc/command.go

```diff
@@ -163,7 +169,7 @@ const (
 	EcdsaSecp256k1Signature2019 = "EcdsaSecp256k1Signature2019"
 )
 
-func addProof(provableData provable, privKey []byte, opts *ProofOptions) error {
+func addProof(provableData provable, privKey []byte, opts *ProofOptions, loader *ld.DocumentLoader) error {
```

Comment:
```suggestion
func (f *Framework) addProof(provableData provable, privKey []byte, opts *ProofOptions) error {
```
You can also embed this function to the `Framework`